chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestDescription.java

```diff
@@ -0,0 +1,104 @@
+package com.griddynamics.jagger.user.test.configurations;
+
+/**
+ * The class contains description of a test.
+ * Here a user can set end points and queries for a test.
+ */
+public class JTestDescription {
+
+    private String id;
+    private String description;
```

Comment:
I would say that "description" naming is quite misleading here, the user may get lost in what do we mean by "description". I would suggest to rename the field into "name", "note"  or "comment".